tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -241,7 +241,29 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args, const char *Argv
 
   const auto &MainFile = M->getMainFile(FileUnitKind::SerializedAST);
   llvm::errs() << "Emitting symbol graph for module file: " << MainFile.getModuleDefiningPath() << '\n';
+  
+  int Success = symbolgraphgen::emitSymbolGraphForModule(M, Options);
+  
+  // Look for cross-import overlays that the given module imports.
+  
+  // Clear out the diagnostic printer before looking for cross-import overlay modules,
+  // since some SDK modules can cause errors in the getModuleByName() call. The call
+  // itself will properly return nullptr after this failure, so for our purposes we
+  // don't need to print these errors.
+  CI.getDiags().removeConsumer(DiagPrinter);
```

Comment:
Nit: I think `CI.removeDiagnosticConsumer()` will do the same thing without `getDiags()`.

Reply:
Interesting! I called it like this to echo an existing call earlier in the function, but i can remove the diagnostic consumer like this if it's easier/faster.